chocolatey/choco-theme
# (#415)(#416)(#417)(#418) choco-theme 0.8.1

## Description Of Changes
* Upgrades ESLint to 9.x
* Fixes PT Sans font path for docs
* Adds scanning of all dependencies for security vulnerabilities
* Upgrades miscellaneous dependencies and upgrades to choco-theme 0.8.1

## Motivation and Context
There were a few dependencies that needed upgraded to allow the `yarn npm audit` command to pass. We want to ship up to date dependencies and remove any vulnerabilities. In addition, the font path to the docs needed fixed to ensure fonts were shown the same for all users.

## Testing
1. Pull down the PR and build it according to the repository's readme.
2. Ensure the following "works":
    1. Switching from dark to light mode.
    2. If any dropdown navigation on the left side bar, ensure the navigation drops down as it should.
    3. Everything "looks" correct. There should be almost no noticeable changes, except for a few colors in light mode have been updated for accessibility. 
3. If everything looks good, approve the pull request but do not merge. Note that after choco-theme is released, and the repository is updated with the new version, you will be notified to approve again.
4. Change subtask item to "Blocked" in ClickUp with a note that it is approved but awaiting a release of choco-theme.

### Operating Systems Testing
* choco-theme 0.8.1 on Windows 10

## Change Types Made

## Change Checklist

* [ ] credits.md file has been updated

## Related Issue
* #415 
* #416 
* #417 
* #418 
* #421 
* ENGTASKS-4000
